### ISSUE TITLE
Improve teams colour code and re-enble dark mode

### ISSUE THREAD

**Issue by bapairaew** [bapairaew/open-fpl#5]
Some teams have to the same colour codes (e.g. MUN and LIV) which makes it harder for user to distinguish them at a glance. The colour codes are current extracted from https://teamcolorcodes.com/soccer/premier-league-color-codes/ which is introduced this issue and moreover the data is. already outdated.

Might be better to just hard code these values base on their current year kit colour since they are changed only once a year.

---

Also somewhat related issue: dark mode was disabled because FPL colour codes and team colour codes worked quite baldly with base Chakra UI dark theme. Since FPL colour codes should be simple to adjust so once team colour is fixed this should be very trivial to fix. If not, create another issue to enable the dark mode.
